src/vs/workbench/contrib/terminal/browser/terminalEditor.ts

```diff
@@ -166,8 +166,7 @@ export class TerminalEditor extends EditorPane {
 			case TerminalCommandId.CreateTerminalEditor: {
 				if (action instanceof MenuItemAction) {
 					const location = { viewColumn: ACTIVE_GROUP };
-					const actions = getTerminalActionBarArgs(location, this._terminalProfileService.availableProfiles, this._getDefaultProfileName(), this._terminalProfileService.contributedProfiles, this._terminalService, this._dropdownMenu);
-					this._registerDisposableActions(actions.dropdownAction, actions.dropdownMenuActions);
+					const actions = getTerminalActionBarArgs(location, this._terminalProfileService.availableProfiles, this._getDefaultProfileName(), this._terminalProfileService.contributedProfiles, this._terminalService, this._dropdownMenu, this._disposableStore);
```

Comment:
Can this be called multiple times for a simple terminal editor? If so we can't just register them against the terminal editor.